lib/ansible/modules/cloud/google/gcp_dns_managed_zone.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017 Google
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by ansible-codegen and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file in README.md and
+#     CONTRIBUTING.md located at the root of this package.
+#
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+################################################################################
+# Documentation
+################################################################################
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: gcp_dns_managed_zone
+description:
+    - A zone is a subtree of the DNS namespace under one administrative
+      responsibility. A ManagedZone is a resource that represents a DNS zone
+      hosted by the Cloud DNS service.
+short_description: Creates a GCP ManagedZone
+version_added: 2.5
+author: Google Inc. (@googlecloudplatform)
+requirements:
+    - python >= 2.7
+options:
+    state:
+        description:
+            - Whether the given object should exist in GCP
+        required: false
```

Comment:
You could update the generator to not add `required: false` lines as that's the default.

Reply:
Done. This was actually a typo.